Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
Wait a second :) How are the normal, non-rboot, applications going to be linked if the memory section is gone? Maybe you should have: one rboot file with memory section, one non-rboot with memory section and one common ld file to be included from the other two.